connectivity/mbedtls/tools/importer/Makefile

```diff
@@ -55,7 +55,7 @@ rsync:
 	#
 	# Copying mbed TLS into mbed library...
 	rm -rf $(TARGET_SRC)
-	rsync -a --exclude='*.txt' $(MBED_TLS_DIR)/library/ $(TARGET_SRC)
+	rsync -a --include='*.[hc]' --exclude='*' $(MBED_TLS_DIR)/library/ $(TARGET_SRC)
```

Comment:
Note: both `--include` and `--exclude` are needed. This means "exclude all files except for .h and .c files".